src/main/java/com/lpvs/controller/LPVSWebController.java

```diff
@@ -0,0 +1,54 @@
+package com.lpvs.controller;
+
+import com.lpvs.entity.LPVSLoginMember;
+import com.lpvs.entity.LPVSMember;
+import com.lpvs.repository.LPVSDetectedLicenseRepository;
+import com.lpvs.repository.LPVSLicenseRepository;
+import com.lpvs.repository.LPVSMemberRepository;
+import com.lpvs.repository.LPVSPullRequestRepository;
+import com.lpvs.service.LPVSLoginCheckService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+@Slf4j
```

Comment:
LPVSWebController.log is not used

Reply:
import lombok.extern.slf4j.Slf4j;
@Slf4j

can be removed

Reply:
Done.